db/migrate/20250425161622_remove_record_source_id_from_ite_client_tables.rb

```
@@ -0,0 +1,5 @@
+class RemoveRecordSourceIdFromIteClientTables < ActiveRecord::Migration[7.0]
```

RuboCop (1.75.5)

[convention] Missing top-level documentation comment for `class RemoveRecordSourceIdFromIteClientTables`.

(Style/Documentation)